package.json

```diff
@@ -10,12 +10,12 @@
     "codecov": "^1.0.1",
     "compass-vertical-rhythm": "^1.3.1",
     "eslint": "^3.5.0",
-    "eslint-config-airbnb": "^11.1.0",
+    "eslint-config-airbnb": "^12.0.0",
     "eslint-plugin-flow-vars": "^0.5.0",
-    "eslint-plugin-import": "^1.15.0",
+    "eslint-plugin-import": "^2.0.1",
```

Comment:
I think this should stay at `1.16.0` until the airbnb config supports it

If you look at the `package.json` from the [eslint-config-airbnb@12.0.0](https://github.com/airbnb/javascript/blob/eslint-config-airbnb-v12.0.0/packages/eslint-config-airbnb/package.json#L66) release it has a peer dependancy of `eslint-plugin-import@1.16.0`